src/Tools/Source/RunTests/TestRunner.cs

```diff
@@ -156,8 +161,6 @@ private void Print(List<TestResult> testResults)
                 errorOutput = string.Format("Command: {0} {1}", _xunitConsolePath, builder.ToString())
                     + Environment.NewLine
                     + errorOutput;
-
-                Process.Start(resultsPath);
```

Comment:
@jaredpar I removed this because it looked odd and didn't make sense.  Am I missing something?